libs/commons/common2.ml

```diff
@@ -3042,24 +3042,13 @@ let unixname () =
   let entry = Unix.getpwuid uid in
   entry.Unix.pw_name
 
-(** [dir_contents] returns the paths of all regular files that are
- * contained in [dir]. Each file is a path starting with [dir].
-  *)
-let dir_contents dir =
-  let rec loop result = function
-    | f :: fs when Sys.is_directory f ->
-        Sys.readdir f |> Array.to_list
-        |> List.filter Sys.file_exists
-        |> List.map (Filename.concat f)
-        |> List.append fs |> loop result
-    | f :: fs -> loop (f :: result) fs
-    | [] -> result
-  in
-  loop [] [ dir ]
+let dir_regex = Str.regexp "^[^\\*]*"
```

Comment:
I worry about the portability of this. Can we use `Fpath`?

Reply:
Where would we use Fpath? Not sure what's not portable here

Reply:
Chatted on Slack, no portability issue here